documentation/04_tutorials/10-part-ix-ui-and-combat.html.md

```diff
@@ -134,7 +134,7 @@ This all seems simple enough, but it's actually going to require several compone
 		FlxG.collide(_player, _mWalls);
 		FlxG.overlap(_player, _grpCoins, playerTouchCoin);
 		FlxG.collide(_grpEnemies, _mWalls);
-		checkEnemyVision();
+		checkEnemyVision;
```

Comment:
Hm, that line is supposed to look like this:

https://github.com/HaxeFlixel/flixel-demos/blob/master/Tutorials/TurnBasedRPG/source/PlayState.hx#L145